### ISSUE TITLE
Bosch Room Thermostat II 230V (RBSH-RTH0-ZB-EU)

### ISSUE THREAD

**Issue by aSauerwein** [dresden-elektronik/deconz-rest-plugin#7553]
### Is there already an existing issue for this?

- [X] I have searched the existing issues and there is none for my device

### Product name

Boosch Room Thermostat II 230V (RBSH-RTH0-ZB-EU)

### Manufacturer

Bosch

### Model identifier

RBSH-RTH0-ZB-EU

### Device type to add

Thermostat

### Node info

<img width="361" alt="image" src="https://github.com/dresden-elektronik/deconz-rest-plugin/assets/5446813/e42956cb-a9db-45c8-a42a-914c923b044e">

### Endpoints and clusters

<img width="267" alt="image" src="https://github.com/dresden-elektronik/deconz-rest-plugin/assets/5446813/0efd1c7a-2200-463f-bf42-54339ea596f4">

### Basic

<img width="361" alt="image" src="https://github.com/dresden-elektronik/deconz-rest-plugin/assets/5446813/c69ed6ac-578f-4f26-a915-f2fe93489893">

### Power Configuration
![image](https://github.com/dresden-elektronik/deconz-rest-plugin/assets/5446813/99604fb4-71f9-458e-9870-e775e4338e9d)

### Thermostat
![image](https://github.com/dresden-elektronik/deconz-rest-plugin/assets/5446813/a4fd42e2-fabd-4591-bf93-2beff86a3327)
![image](https://github.com/dresden-elektronik/deconz-rest-plugin/assets/5446813/bf012ebc-4415-46c1-8fe8-ae6b0a41e37c)
![image](https://github.com/dresden-elektronik/deconz-rest-plugin/assets/5446813/9388a27c-66d6-4675-af04-feab6d41f75d)
![image](https://github.com/dresden-elektronik/deconz-rest-plugin/assets/5446813/4270ecb5-eb2f-44ef-ae11-a31d63cdd612)
![image](https://github.com/dresden-elektronik/deconz-rest-plugin/assets/5446813/8d0cdec4-c871-44f7-8709-f40b1fac7baf)
![image](https://github.com/dresden-elektronik/deconz-rest-plugin/assets/5446813/878b8130-bd85-494b-8f7a-0f5da5a24e1a)
![image](https://github.com/dresden-elektronik/deconz-rest-plugin/assets/5446813/6e2e4eb1-6316-499c-a6d2-9680ce817e8a)
![image](https://github.com/dresden-elektronik/deconz-rest-plugin/assets/5446813/ad62fa9f-f684-4f1c-927f-a844cb438b8a)
![image](https://github.com/dresden-elektronik/deconz-rest-plugin/assets/5446813/37f2d4a9-f0b7-451d-ac9b-71d43a951540)

### Thermostat User Interface Configuration
![image](https://github.com/dresden-elektronik/deconz-rest-plugin/assets/5446813/8094c78a-73ac-4e77-b651-20b806c2419f)

### Relative Humidity Measurement
![image](https://github.com/dresden-elektronik/deconz-rest-plugin/assets/5446813/8fe5439f-1185-4112-a3cd-2f616edb2afc)

### Diagnostics
![image](https://github.com/dresden-elektronik/deconz-rest-plugin/assets/5446813/fe62ca56-2ad7-4931-be8b-3fe9254db662)

### Poll Control
![image](https://github.com/dresden-elektronik/deconz-rest-plugin/assets/5446813/7dc555f6-6453-499d-8e31-2f9ef873d961)

**Comment by aSauerwein** [comment]
for testing I copied https://github.com/dresden-elektronik/deconz-rest-plugin/blob/master/devices/bosch/room_thermostat2.json, changed the model id and imported it into deconz. 
So far it looks good, i can read and change the temperature setpoint. 

So I'd assume most of the things can be copied from https://github.com/dresden-elektronik/deconz-rest-plugin/blob/master/devices/bosch/room_thermostat2.json. 

also according the the implemention from zigbe2mqtt it looks very similar
https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/src/devices/bosch.ts

**Comment by Mimiix** [comment]
Can you please add all screenshots?

**Comment by aSauerwein** [comment]
is there a way to make one screenshot of the whole attribute table ? currently it shows only 4 elements without scrolling

**Comment by Mimiix** [comment]
> is there a way to make one screenshot of the whole attribute table ? currently it shows only 4 elements without scrolling

No, not that I know of.  Perhaps you can increase your resolution.

**Comment by aSauerwein** [comment]
just updated the Issue with all screenshots. Let me know if you need more

**Comment by aSauerwein** [comment]
I'm afraid to make a pull request, cause I just copy pasted things from the existing battery power room thermostat. 
but this ddf seems to work fine for me: 
https://gist.github.com/aSauerwein/b5ce7460a78820c8bb6dcdc451118a10

changes: 
* i removed the config/battery
* added state/valve with read&parse of 0x4020

open questions
* I did not understand what the "bindings" are used for
* deconz-gui still shows the battery icon. don't know what needs to be changed to get that icon removed. 
* for state/valve I just used `Item.val = Attr.val` but I'm not sure if that correctly matches the percent valve opening
* I cannot control the thermostat with phoscon web app. It just displays 40F and nothing changes, but it's working fine over homeassistant and rest api
